.github/workflows/test.yaml

```diff
@@ -56,7 +56,7 @@ jobs:
       uses: actions/cache@v2
       with:
         key: ${{ runner.os }}-rustup-${{ hashFiles('rust-toolchain') }}
-        path: '~/.rustup/toolchains/1.52.1-*
+        path: '~/.rustup/toolchains/1.53.0-*
```

Comment:
Does this warrant a comment in the `rust-toolchain` file to tell people to update these paths with the new version?

Reply:
Probably. And perhaps automate it to fail CI if changes were made.